### DISCUSSION TITLE
Activate Long-Term Memory Engine Not Working

### DISCUSSION THREAD

**Discussion by Shariq12345** [pieces-app/support#552] [Q&A]
I was using the Pieces OS for quite sometime but recently i cannot activate the long-term memory. I click on it and it shows a activating loader and nothing happens it resets and again show the same "activate long-term engine" button.

**Comment by mack-at-pieces** [comment]
Hey @Shariq12345 - Noticed you also made an issue for this as well and the team has been in contact over there. I'm going to close this down :)
